app/helpers/admin_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AdminHelper
-  def thing_type(thing)
-    request.path.split('admin/').last.split("/").first.capitalize.singularize
+  def thing_type(_thing)
```

Comment:
At the time I was focusing more on testing than changing code, but it's a good idea to take it out.